examples/resource_partitioner/async_customization.cpp

```diff
@@ -407,7 +407,7 @@ int test(Executor &exec)
     auto fd = dataflow(exec,
         [](future<uint16_t> && f1, future<double> && f2)
```

Comment:
Please use `std::uint16_t` here as well (and similarly for all other sized integral types).

Reply:
done. And force pushed again. sorry for missing it first time around